CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
     - The Mutable reverb model has been set as the default reverb model for new songs. Old songs will respect the reverb model used with those songs.
 - Fixed stereo unison spread + ringmod + noise causing excessively loud output.
 - Fixed some bugs around the waveform Loop Lock feature which allowed setting invalid loop points.
+- Added compressors to synths, kits, audio clips, and kit rows
```

Comment:
Should include a short note about how to activate the feature, and be sorted with the rest of the "added" lines at the beginning.